app/models/project.rb

```diff
@@ -21,7 +21,7 @@ class Project < ApplicationRecord
   validates :establishment_sharepoint_link, presence: true, url: {hostnames: SHAREPOINT_URLS}, on: :create
   validates :trust_sharepoint_link, presence: true, url: {hostnames: SHAREPOINT_URLS}, on: :create
 
-  validate :establishment_exists, :trust_exists, on: :create
+  validate :establishment_exists, :trust_exists, on: :create, if: proc { |project| project.errors.empty? }
```

Comment:
Is there a reason we don't make this about the values themselves rather than errors? Something like:

```
  validate :trust_exists, on: :create, unless: proc { |project| project.incoming_trust_ukprn.nil? }
  validate :establishment_exists, on: :create, unless: proc { |project| project.urn.nil? }
```

Reply:
Done 👍